server/app/views/BaseHtmlLayout.java

```diff
@@ -151,10 +152,20 @@ public Content render(HtmlBundle bundle) {
       bundle.setTitle(String.format("%s — %s", currentTitle, getTitleSuffix()));
     }
     // Best practice: add ❤️ every time you touch this file :)
-    bundle.addMetadata(meta().withName("thanks").withContent("Thank you Bion ❤️❤️"));
+    bundle.addMetadata(meta().withName("thanks").withContent("Thank you Bion ❤️❤️❤️"));
     return bundle.render();
   }
 
+  protected void maybeAddSessionTimeoutModals(HtmlBundle bundle, Messages messages) {
```

Comment:
```suggestion
  protected void addSessionTimeoutModals(HtmlBundle bundle, Messages messages) {
```

(blocking) It's really only a maybe if the feature flag is disabled, but that will eventually go away.